### PR TITLE
MODINVOICE-121 - Release 3.0.1

### PR DESCRIPTION
**Pull request by khandramai**
MODINVOICE-121 - Release 3.0.1

### CODE REVIEW

File: ramls/invoice.raml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
 
 title: Invoice
-version: v3.0
+version: v3.1
```
Comment (KVupp) [inline ramls/invoice.raml]: is this version increment necessary? We haven't modified anything in the request/response

Comment (khandramai) [reply ramls/invoice.raml]: In scope of MODINVOICE-120 hardcoded voucher "accounting code" was changed to real accounting code from invoice. This affects voucher creation during invoice approval flow, i.e invoice updating. Is this not a reason for minor version increasing?

Comment (KVupp) [reply ramls/invoice.raml]: yea that is the reason to increase the interface version . But what does it have to do with the raml version?

Comment (khandramai) [reply ramls/invoice.raml]: Yes, correct. I've already reverted. Thanks.

**Pull request merged**